path-manager/app/services/PathStore.scala

```diff
@@ -133,6 +198,21 @@ object PathStore extends Logging {
     }
   }
 
+  def deleteAlias(path: String) = {
+    logger.debug(s"deleting alias records for path [$path]")
+    val pathItems = Option(Dynamo.pathsTable.getItem("path", path)).map(PathRecord(_))
+    val remaining = pathItems map { item =>
+      val path = item.path
+      if (item.`type` == "alias") {
+        logger.debug(s"deleting path [$path] for id [${item.identifier}]")
+        Dynamo.pathsTable.deleteItem("path", path)
+      }
+      item.identifier
+    }
+    logger.debug(s"deleted path records for [$path] successfully")
+    remaining
```

Comment:
I'm not sure this does return the 'remaining' paths... it appears to map every entry in `pathItems` to its `identifier` regardless of whether it has been deleted

Reply:
also the logger message is ambiguous as its possible no paths were deleted at all (and probably not needed given the more explicit one inside the `map`)

Reply:
whoops, good catch

Reply:
Fixed, but probably should just delete.